diff --git a/src/makefrag b/src/makefrag
index 8f85ed2..d7e2d0c 100644
--- a/src/makefrag
+++ b/src/makefrag
@@ -95,9 +95,9 @@ WITH_VOIPSTREAM := $(shell (cd .. && $(FEATURETOOL) -q isenabled VoIPStream && e
 WITH_VISUALIZERS := $(shell (cd .. && $(FEATURETOOL) -q isenabled visualization && echo enabled) )
 ifeq ($(WITH_VISUALIZERS), enabled)
   ifeq ($(WITH_OSG), yes)
-    OMNETPP_LIBS += -losg -losgText -losgDB -losgGA -losgViewer -losgUtil -lOpenThreads # TODO: use $(OSG_LIBS) from Makefile.inc? Does that include -losgText? Why not?
+    OMNETPP_LIBS += $(OSG_LIBS) -losgText
     ifeq ($(WITH_OSGEARTH), yes)
-      OMNETPP_LIBS += -losgEarth -losgEarthUtil # TODO: use $(OSGEARTH_LIBS) from Makefile.inc? -lgeos_c might also be needed. Is that included? Should it be?
+      OMNETPP_LIBS += $(OSGEARTH_LIBS)
     endif
   endif
 endif
